packages/components/src/font-size-picker/test/index.js

```diff
@@ -203,9 +203,9 @@ describe( 'FontSizePicker', () => {
 				);
 				const element = screen.getByLabelText( 'Large' );
 				expect( element ).toBeInTheDocument();
-				expect( element.children[ 0 ].textContent ).toBe( '1.7' );
+				expect( element.children[ 0 ].textContent ).toBe( 'L' );
 			} );
-			it( 'should use incremental sequence of numbers as labels if we have complex css', () => {
+			it( 'should use incremental sequence of t-shirt as labels if we have complex css', () => {
```

Comment:
```suggestion
			it( 'should use incremental sequence of t-shirt sizes as labels if we have complex css', () => {
```
Small wording improvement.